.env.default

```diff
@@ -13,3 +13,6 @@ export TEST_DATABASE_HOST=
 export TEST_DATABASE_NAME=
 export TEST_DATABASE_USER=
 export TEST_DATABASE_USER_PASSWORD=
+
+
+export BRAINSTORMING_TEASERS=true
```

Comment:
Lets namespace or scope the env variables, e.g. `MW_FEATURE_BRAINSTORMING_TEASER`